packages/cli/templates/webcomponents/igc-ts/projects/_base/files/vite.config.ts

```diff
@@ -21,10 +33,22 @@ export default defineConfig({
     chunkSizeWarningLimit: 10 * 1024 * 1024 // 10 MB
   },
   plugins: [
+    {
+      name: 'replace-grid-css-paths',
+      apply: 'build',
+      transform(code, id) {
+        if (id.endsWith('.js')) {
+          themeFiles.forEach(t => { code = code.replaceAll(t, `../../${path.basename(t)}`); });
+          return code;
+        }
+      }
+    },
     /** Copy static assets */
     viteStaticCopy({
       targets: [
         { src: 'src/assets', dest: 'src' },
+        { src: 'ig-theme.css', dest: '' },
```

Comment:
I understand why css from packages needs a static copy, but why the ig-theme which is part of the source. Is that the way vite instructs css to be handled, because I think this can also just be imported tp get it included.

Reply:
Wait.. that's not even it the project files? :O

Is that part for the CodeGen? Not sure it even belongs here if so, though you can argue it's functionality to allow to set global styling for all views; hint: the current `styles.css` is useless for that and your `img, video` styles and whatever missing resets we apply in other projects that are missing won't work from that either.

To that end, `ig-theme` can and should be in the templates with a comment explaining it sets up styles for all the views. The name also shouldn't really be `ig-theme` (that file actually had a different purpose initially) and should be included in all views generated by the CLI ofc.
Then you can justify the file handling in the project ;D